exams/pearson/upload.py

```diff
@@ -28,6 +28,10 @@ def get_connection():
         if getattr(settings, key) is None:
             missing_settings.append(key)
 
+    pearson_exams_sync = settings.FEATURES.get("PEARSON_EXAMS_SYNC", False)
```

Comment:
This will result in a logged exception and the point of adding this feature flag was to avoid the exception in the first place. Could you please remove this and add only these lines to the beginning of both tasks:

```python
if not settings.FEATURES.get("PEARSON_EXAMS_SYNC", False):
    return
```

Make sure to add tests to ensure that the body of the function is / is not being run depending on the value of this flag.